site/content/community/proposals.adoc

```diff
@@ -17,40 +17,21 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-linkTitle: 'Proposals'
-title: 'Apache Polaris Proposals'
+linkTitle: 'Proposals & Roadmap'
+title: 'Apache Polaris Proposals & Roadmap'
 weight: 200
 type: docs
-# TODO remove when adding the first proposal
-toc_hide: true
+toc_hide: false
 hide_summary: true
 ---
 
-== Active Proposals
+== Proposals
 
-[cols="4,2,4,4"]
-|===
-| Topic | Date Created | Proposal Document | Dev-list discussion
+* [Active](https://github.com/apache/polaris/issues?q=is%3Aissue%20state%3Aopen%20label%3Aproposal)
+* [Past](https://github.com/apache/polaris/issues?q=is%3Aissue%20state%3Aclosed%20label%3Aproposal)
 
-| Sample topic placeholder
-| 2024/09/06
-| Link to Google Doc
-| Link to dev-mailing list discussion
 
-|===
+== Polaris Roadmap
 
-
-== Previous Proposals
-
-
-[cols="4,2,4,4,2"]
-|===
-| Topic | Date Created | Proposal Document | Dev-list discussion + vote | Resolution
-
-| Sample topic placeholder
-| 2024/09/06
-| Link to Google Doc
-| Link to dev-mailing list discussion
-| Accepted
-
-|===
+* [Discussed Roadmap](https://github.com/apache/polaris/discussions/1028)
+* [Kanban Board](https://github.com/orgs/polaris-catalog/projects/1)
```

Comment:
Do we need this Kanban board? Given that we will also create individual project for each feature.

Reply:
Good point. Let me remove it.